docs/index.rst

```diff
@@ -194,8 +194,8 @@ class to define a business object, such as - ShoppingBag::
         protected function init(): void {
             parent::init();
 
-            $this->hasOne('user_id', new User());
-            $this->hasMany('Items', new Item())
+            $this->addHasOne('user_id', new User());
```

Comment:
these changes shouldn't be in this PR

Reply:
I updated the docs here to match the latest changes, there should not be any of these in atk4/core actually :D